poc/vdaf_poc/idpf_bbcggi21.py

```diff
@@ -24,10 +24,9 @@
 # types, which aids with self-documentation.
 
 FieldVec: TypeAlias = list[Field64] | list[Field255]
-CorrectionWordTuple: TypeAlias = tuple[bytes, tuple[Field2, Field2], FieldVec]
+CorrectionWord: TypeAlias = tuple[bytes, tuple[Field2, Field2], FieldVec]
```

Comment:
@divergentdave I can revert this if you like. CorrectionWord seems just as accurate to me and is a bit pithier.